examples/freertos/FreeRTOSConfig.h

```diff
@@ -44,6 +44,12 @@
 #include <stdint.h>
 extern uint32_t SystemCoreClock;
 
+#if defined STM32F1
+	// This is enough for Blinky example.
+	#define configMINIMAL_STACK_SIZE		( ( unsigned short ) 50 )
+	#define configTOTAL_HEAP_SIZE			( ( size_t ) ( 2 * 1024 ) )
```

Comment:
If the example works with these changes on F1 there is no reason for it not to work on other platforms. Maybe Just apply these settings for all boards (to keep code easy)

Reply:
I did that change, tested on my STM32VLDISCOVERY, but I have no other boards to make tests